pyproject.toml

```diff
@@ -0,0 +1,106 @@
+[build-system]
+requires = [
+    "setuptools >= 65",
+    "setuptools_scm[toml] >= 6.2",
+    "setuptools_scm_git_archive",
```

Comment:
`setuptools-scm` is recommending just this nowadays:

```
[build-system]
requires = ["setuptools>=64", "setuptools_scm>=8"]
build-backend = "setuptools.build_meta"
```

At some point the optional `[toml]` bits became standard in `setuptools-scm`, and in version 8 adding `[toml]` doesn't do anything. If they drop this optional dependencies block in version 9, this might cause issues.